library/core/src/ops/range.rs

```diff
@@ -335,6 +335,7 @@ impl<Idx: PartialOrd<Idx>> RangeTo<Idx> {
 /// assert_eq!(arr[1..=3], [   1, 2, 3   ]); // This is a `RangeInclusive`
 /// ```
 #[lang = "RangeInclusive"]
+#[doc(alias = "..")]
```

Comment:
This shouldn't be on RangeInclusive.

Reply:
The idea is that it’s allowing searching for the prefix `..` that’s in common for both `..` and `..=`. Similar to how [`+` finds `AddAssign`](https://doc.rust-lang.org/std/?search=%2B) even though that’s `+=` as an operator.